examples/ionic-angular/README.md

```diff
@@ -0,0 +1,78 @@
+# Ionic Angular Conference Application
```

Comment:
Similarly, I think this readme needs to be updated to match this pattern:

https://github.com/zeit/now/blob/master/examples/hugo/README.md